### PR TITLE
VFB-213: Changes to shopping and shipping list

### PR DESCRIPTION
**Pull request by kaydentsoftwire**
## What's changed
Shopping list icon for collection is now feet
I also added a gap between Shopping List and | as this seems to be consistent with what we're changing elsewhere
The ticket requirements for the shipping labels were covered by another ticket so no longer included in this one

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image | image |

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

### CODE REVIEW

File: src/pdf/ShoppingList/ShoppingListPdf.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ const styles = StyleSheet.create({
     title: {
         fontSize: "22pt",
         fontFamily: "Helvetica-Bold",
+        paddingRight: "6pt",
     },
```
Comment (kaydentsoftwire) [inline src/pdf/ShoppingList/ShoppingListPdf.tsx:78]: adding gap between title and | (and also works on | and the icon)

File: src/pdf/ShoppingList/ShoppingListPdf.tsx [hunk 2]
```diff
@@ -226,12 +227,12 @@ const SingleShoppingList: React.FC<SingleShoppingListProps> = ({ parcelData }) =
...
                             <Text style={styles.title}>Shopping List</Text>
-                            <Text style={styles.title}> |</Text>
+                            <Text style={styles.title}>|</Text>
                             <FontAwesomeIconPdfComponent
                                 faIcon={
                                     parcelData.parcelInfo.collectionSite === "N/A - Delivery"
                                         ? faTruck
-                                        : faBuildingCircleArrowRight
+                                        : faShoePrints
                                 }
```
Comment (kaydentsoftwire) [inline src/pdf/ShoppingList/ShoppingListPdf.tsx:236]: changing the icon displayed

**Review by AmeliaIsCoding — APPROVED** [review]
LGTM

**Review by 8kristy — APPROVED** [review]
> Shopping list icon for **deliveries** is now feet

I guess that's just a typo given that the code seems to have updated collection and that's what we want to change :) Otherwise all good

**Comment by kaydentsoftwire** [comment]
> > Shopping list icon for **deliveries** is now feet
> 
> I guess that's just a typo given that the code seems to have updated collection and that's what we want to change :) Otherwise all good

oh whoops that's right! Thanks :)

**Pull request merged**